docs/development/core/public/kibana-plugin-public.chromenavlink.disablesuburltracking.md

```diff
@@ -1,17 +1,17 @@
-<!-- Do not edit this file. It is automatically generated by API Documenter. -->
-
-[Home](./index.md) &gt; [kibana-plugin-public](./kibana-plugin-public.md) &gt; [ChromeNavLink](./kibana-plugin-public.chromenavlink.md) &gt; [disableSubUrlTracking](./kibana-plugin-public.chromenavlink.disablesuburltracking.md)
-
-## ChromeNavLink.disableSubUrlTracking property
-
-> Warning: This API is now obsolete.
-> 
-> 
-
-A flag that tells legacy chrome to ignore the link when tracking sub-urls
-
-<b>Signature:</b>
-
-```typescript
-readonly disableSubUrlTracking?: boolean;
-```
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
```

Comment:
I think you may have ran the update script here based on an older version of master. We've been battling with getting these files to use the same EOL character. After merging master, can you run this again and commit any changes? `node scripts/check_core_api_changes --docs`

Reply:
No changes after merging master running the script